tests/containers/award/AwardContainer-test.jsx

```diff
@@ -365,105 +367,231 @@ const txnData = {
+        // Mocking an AwardSummary result
+        const expectedAwardData = {
+            _jsid: "award-2",
+            award_id: "VA760PPVFY2015OCT",
+            award_type: "Delivery Order",
+            awarding_agency_name: "VETERANS AFFAIRS, DEPARTMENT OF",
+            awarding_office_name: "",
+            awarding_subtier_name: "VETERANS AFFAIRS, DEPARTMENT OF",
+            date_signed: "Invalid date",
+            description: "EXPRESS REPORT PHARMACY PRIME VENDOR VA760PPVFY2015OCT",
+            funding_agency_name: "VETERANS AFFAIRS, DEPARTMENT OF",
+            funding_office_name: "",
+            funding_subtier_name: "VETERANS AFFAIRS, DEPARTMENT OF",
+            id: 8533,
+            latest_transaction: null,
+            parent_award: 7758,
+            period_of_performance_current_end_date: "Invalid date",
+            period_of_performance_start_date: "Invalid date",
+            pop_city: undefined,
+            pop_state_province: "",
+            pop_zip: undefined,
+            potential_total_value_of_award: "$227,269,217",
+            recipient_business_type: "Unknown Business Type",
+            recipient_city: "San Francisco",
+            recipient_country: "UNITED STATES",
+            recipient_duns: "177667227",
+            recipient_name: "MCKESSON CORPORATION",
+            recipient_parent_duns: "177667227",
+            recipient_state_province: "CA",
+            recipient_street: "ONE POST ST",
+            recipient_zip_postal: "94104",
+            total_obligation: "$227,269,217",
+            type: "Delivery Order",
+            type_description: "Delivery Order",
+            type_of_contract_pricing: "J",
+            type_of_contract_pricing_description: "Firm Fixed Price"
+        };
+
+        // Mock the call and validate it matches the expected award summary
+        const mockReduxAward = jest.fn((args) => {
+            expect(args).toEqual(expectedAwardData);
+        });
         // Set up container with mocked award action
         const awardContainer = shallow(
             <AwardContainer
                 params={parameters}
                 award={award}
-                setSelectedAward={AwardActions.setSelectedAward} />);
+                setSelectedAward={mockReduxAward} />);
 
         const parseAwardSpy = sinon.spy(awardContainer.instance(),
             'parseAward');
 
-        // Parse award
-        awardContainer.instance().parseAward(fullData);
+        // Run function on instance of container
+        awardContainer.instance().parseAward(awardData);
 
         // checking that it ran
         expect(parseAwardSpy.callCount).toEqual(1);
+        expect(mockReduxAward).toHaveBeenCalledTimes(1);
 
         // reset the spies
         parseAwardSpy.reset();
     });
 
     // parse transactions
     it('should parse the returned transactions and send to redux store', () => {
+        const expectedTxnData = [{
+            _jsid: "contract-transaction-3",
```

Comment:
`_jsid` is a unique value which is controlled by the lodash library, so I'm not sure we can be certain what to expect from it. To avoid the test randomly breaking, I'd suggest excluding `_jsid` from the expected value and deleting the `_jsid` property from `args`  (`delete args._jsid`).